cmis/cmis_backend.py

```diff
@@ -37,37 +39,40 @@ class cmis_backend(orm.Model):
             lambda self, *a, **kw: self._select_versions(*a, **kw),
             string='Version',
             required=True,
+            help="Version",
```

Comment:
@guewen: it is just the cmis version. Maybe, it is better to put "Cmis version" in the help